### PR TITLE
Check for Websockets in Fallback functional tests

### PR DESCRIPTION
**Pull request by mikaelm12**
Issue: https://github.com/aspnet/SignalR/issues/1629

This should fix the failures in this this functional test.

### CODE REVIEW

File: test/Microsoft.AspNetCore.SignalR.Tests/EndToEndTests.cs [hunk 1]
```diff
@@ -404,6 +414,23 @@ public Task StopAsync()
             }
         }
 
+        private static bool IsWebSocketsSupported()
```
Comment (JamesNK) [inline test/Microsoft.AspNetCore.SignalR.Tests/EndToEndTests.cs]: Lets move this to `Microsoft.AspNetCore.SignalR.Tests.Utils` and reuse it everywhere (including that conditional test attribute)

Comment (mikaelm12) [reply test/Microsoft.AspNetCore.SignalR.Tests/EndToEndTests.cs]: Yeah, probably about time to move this

Comment (JamesNK) [reply test/Microsoft.AspNetCore.SignalR.Tests/EndToEndTests.cs]: In fact I can see a `TestHelpers.IsWebSocketsSupported()` method already exists in that project. Check with @anurse to see whether this logic is better than what is already in that method and possibly replace it.

Comment (mikaelm12) [reply test/Microsoft.AspNetCore.SignalR.Tests/EndToEndTests.cs]: Yeah, I wen't ahead and replaced the logic. I can reverted it though. Thoughts? @davidfowl  @anurse

Comment (BrennanConroy) [reply test/Microsoft.AspNetCore.SignalR.Tests/EndToEndTests.cs]: Remove

Comment (anurse) [reply test/Microsoft.AspNetCore.SignalR.Tests/EndToEndTests.cs]: I think the new logic is good. This copy isn't needed though right?

**Pull request merged**